packages/nextjs/src/config/webpack.ts

```diff
@@ -100,6 +101,22 @@ export function constructWebpackConfigFunction(
           ],
         });
       }
+
+      // Prevent `@vercel/nft` (which nextjs uses to determine which files are needed when packaging up a lambda) from
+      // including any of our build-time code or dependencies. (Otherwise it'll include files like this one and even the
+      // entirety of rollup and sucrase.)
+      const nftPlugin = newConfig.plugins?.find((plugin: WebpackPluginInstance) => {
+        const proto = Object.getPrototypeOf(plugin) as WebpackPluginInstance;
+        return proto.constructor.name === 'TraceEntryPointsPlugin';
+      }) as WebpackPluginInstance & { excludeFiles: string[] };
+      if (nftPlugin) {
+        nftPlugin.excludeFiles.push(path.join(__dirname, 'withSentryConfig.js'));
```

Comment:
m: Should we be a bit more defensive here in case `excludeFiles` is `undefined` or whatever, or when Next.js changes stuff in future updates? Maybe we wrap this in a try-catch?

Reply:
Fair point. Done.

Reply:
Is it pushed?

Reply:
Yup. Line 115.

```js
nftPlugin.excludeFiles = nftPlugin.excludeFiles || [];	
```